src/ext/property-panel/settings.js

```diff
@@ -255,11 +255,11 @@ const getChartExploration = (env) =>
 
 const getUsePagination = ({ flags }) =>
   flags.isEnabled('PS_18291_SN_TABLE_BASIC_FEATURES') && {
-    ref: 'presentation.usePagination',
+    ref: 'usePagination',
```

Comment:
I am curious. Why do we decide to remove presentation?

Reply:
The other presentation properties are on the root, so it made no sense to only have this prop there

Reply:
okay, it makes sense to me. Thx